sdk/management/azure/pom.xml

```diff
@@ -16,7 +16,7 @@
   <packaging>jar</packaging>
 
   <name>Microsoft Azure SDK</name>
-  <description>This package contains Microsoft Azure SDK.</description>
+  <description>This package contains Microsoft Azure SDK. Reference https://aka.ms/azure-sdk-java-mgmt</description>
```

Comment:
This is probably not a best wording. Any suggestion?

Don't want to call it doc because it is not, don't want to call it project or repo because it seems not appropriate...

Reply:
Could we add it in the url below?

Reply:
For more details, please visit?

Reply:
For information on how to use this package, please see [here](link)

Reply:
> Could we add it in the url below?

Since there is only one url, not sure if it is appropriate to replace it.

Reply:
> For more details, please visit?

Eh, thought about it, but it seems not saying anything useful, except giving the link...